subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/nativeplatform/SwiftCleanBuildPerformanceTest.groovy

```diff
@@ -19,15 +19,18 @@ package org.gradle.performance.regression.nativeplatform
 import org.gradle.initialization.ParallelismBuildOptions
 import org.gradle.performance.AbstractCrossVersionGradleProfilerPerformanceTest
 import org.gradle.performance.categories.SlowPerformanceRegressionTest
+import org.gradle.util.Requires
+import org.gradle.util.TestPrecondition
 import org.junit.experimental.categories.Category
 import spock.lang.Unroll
 
 @Category(SlowPerformanceRegressionTest)
+@Requires(TestPrecondition.NOT_WINDOWS)
```

Comment:
I think it would be better to say that this is supposed to work on Linux and macOS.